src/Jackett.Common/Definitions/animeworld.yml

```diff
@@ -1,49 +1,32 @@
 ---
 id: animeworld
-name: AnimeWorld
-description: "AnimeWorld (AST4u) is a GERMAN Private site for TV / MOVIES / ANIME / HENTAI"
+name: Animeworld
+description: "AnimeWorld (AW) is a GERMAN Private site for ANIME / MANGA / HENTAI"
 language: de-de
 type: private
-encoding: iso-8859-1
+encoding: UTF-8
 links:
   - https://animeworld.cx/
```

Comment:
https://animeworld.cx/ currently returning ERR_CONNECTION_REFUSED for me
Is the site filtering only German IP addresses or is the site down or moved to a new domain?

Reply:
No, but they had open Signup which got posted on reddit and the someone DDosed the site. It should now be resolved.